src/GeekPlatform/ViewModels/Jelentkezes/JelekezesViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using GeekPlatform.Models;
+
+namespace GeekPlatform.ViewModels.Jelentkezes
+{
+    public class JelekezesViewModel : ViewModelBase
+    {
+        public JelekezesViewModel(IEnumerable<Course> courselist)
+        {
+            Courses = courselist.Select(x => new CourseViewModel(x)).ToList();
```

Comment:
Utóbbi egyszerűbb, szóval most mindenképp az lesz, plusz alapból is logikus az is mert közvetlen felette van egy "kurzusaim" menüpont ahol pont a fordított szűrő lesz.